kep/454-numa-nodes-scoring/README.md

```diff
@@ -47,6 +47,14 @@ to get available resources on the worker nodes and identify which topology polic
 For now available ScoringStrategies are running only for `single-numa-node` TopologyManager policy.
 `LeastNUMANodes` strategy can score nodes running all TopologyManager policies.
 
+Kubernetes `v1.26` introduced `TopologyManagerPolicyOptions` that allows to control `TopologyManager` behaviour.
+New option `prefer-closest-numa-nodes` will favor sets of NUMA nodes with shorter distance between them when making admission decisions.
+`LeastNUMANodes` ScoringStrategy will also take distance between topology zones into consideration and favor nodes with the smallest average distance between zones.
+Distances between topology zones are exposed as `Costs` field in [NRT CRD](https://github.com/k8stopologyawareschedwg/noderesourcetopology-api/blob/master/pkg/apis/topology/v1alpha1/types.go#L56).
+This behaviour will be enabled by default, since [CRD][1] doesn't expose `TopologyManagerPolicyOptions`,
```

Comment:
nit: ...since the CRD...

Reply:
fixed

Reply:
Thanks! for the future, no need from my side to re-upload *just* for so tiny NITs.